keyboards/electronlab/klor/keyboard.json

```diff
@@ -0,0 +1,381 @@
+{
+    "manufacturer": "ElectronLab",
+    "keyboard_name": "KLOR",
+    "maintainer": "Lefuneste83",
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "development_board": "elite_pi",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP28", "pin_b": "GP29", "resolution": 2}
+        ]
+    },
+    "tapping": {
+        "term": 250
+    },
+    "combo": {
+        "term": 150
+    },
+    "features": {
+        "audio": true,
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "haptic": true,
+        "mousekey": true,
+        "oled": true,
```

Comment:
If OLEDs are enabled at the keyboard level, you need some code that utilises them at the keyboard level as well, in `klor.c`.

Reply:
I've put back the enable test for hardware features in config.h at keyboard level